scripts/test.sh

```diff
@@ -646,16 +646,26 @@ function run_pass {
   shift 1
   log_callout -e "\\n'${pass}' started at $(date)"
   if "${pass}_pass" "$@" ; then
-    log_success "'${pass}' completed at $(date)"
+    log_success "'${pass}' PASSED and completed at $(date)"
+    return 0
   else
-    log_error "FAIL: '${pass}' failed at $(date)"
-    exit 255
+    log_error "FAIL: '${pass}' FAILED and completed at $(date)"
+    return 2
```

Comment:
Is it possible to make this gated by environment variables so you can turn it on in your environment?

Reply:
Great suggestion, this approach sounds good to me.

Reply:
I have added the gate by the variable `PASSES_CONTINUE`.
Can you please check now?